src/Jecoute/FileStatsService.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Jecoute;
+
+use AppBundle\Entity\Jecoute\Survey;
+use AppBundle\Entity\Jecoute\SurveyQuestion;
+use AppBundle\Repository\Jecoute\DataAnswerRepository;
+
+class FileStatsService
```

Comment:
What do you think to rename this class by `StatisticsExporter` ?

Reply:
renamed